src/components/StaffList/StaffList.scss

```diff
@@ -0,0 +1,53 @@
+$light-gray: #aaa;
+$medium-gray: #ddd;
```

Comment:
These are in the ```_variables.scss``` partial so you can delete from here.

Reply:
oops, that was a boo-boo